uniswap-v2/src/abi.rs

```diff
@@ -1,15 +1,15 @@
 // DO NOT EDIT - the file is generated by build script
 #[rustfmt::skip]
 #[allow(unused_imports)]
-#[path = "../target/abi/erc20.rs"]
-pub mod erc20;
+#[path = "../target/abi/ERC20.rs"]
+pub mod ERC20;
```

Comment:
We are getting warning here's for not following the standard syntax. Would it be possible to change this back to snakecase? (We can keep the filename uppercase if you like though?)

Reply:
Can we keep this as it is? We can probably add `#[allow(non_snake_case)]` to allow non snake case names in codegen.rs

Reply:
Ok sure we can do that then